bin/autoproj_bootstrap

```diff
@@ -601,4 +601,4 @@ if !ops.skip_stage2?
     ops.call_stage2
 end
 ops.run_autoproj 'bootstrap', *bootstrap_options
```

Comment:
I would prefer that you change `#run_autoproj` to raise if the call fails (`Kernel#system` returns true/false on success/failure)